ecommerce/values-dev.yaml

```
@@ -0,0 +1,11 @@
+# Development environment overrides for ecommerce chart
+# Keep this file minimal; set only values that differ from values.yaml
+replicaCount: 1
+
+# Development environment overrides for ecommerce chart
+# Keep this file minimal; set only values that differ from values.yaml
+replicaCount: 1
```

YAMLlint (1.37.1)

[error] duplication of key "replicaCount" in mapping

(key-duplicates)